src/molecules/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Toolbar from '@material-ui/core/Toolbar';
+import { StyledNavBar } from './styles';
+
+const NavBar: React.FC = () => {
+  return (
+    <StyledNavBar position="static">
+      <Toolbar>
+        <h2>VikeLabs / </h2>
```

Comment:
Can we change these to both be the same size? I think it looks best when they're both h3's but I'll get someone else to take a look at it as well. 

Can we reduce the margin a little bit? I think the navbar is just a bit too tall. 

Also can you link to https://vikelabs.ca/